client/src/style/scss/overrides.scss

```diff
@@ -199,6 +199,12 @@ table.info_data_table th:nth-child(1) {
         cursor: default;
 
         // differentiate disabled items more
-        color: #bfc7cf !important;
+        color: #bfc7cf;
```

Comment:
While on it can you replace this fixed color with one of the theme colors in blue.css.

Reply:
I don't think we have one that fits, but I made it dependent on the theme, instead of hard coding it.
Would it maybe make sense to include it in the theme instead?